cmd/verify/verify.md

```diff
@@ -0,0 +1,48 @@
+# sda-pipeline: verify
+
+Uses a crypt4gh secret key, this service can decrypt the stored files and checksum them against the embedded checksum for the unencrypted file.
+
+## Service Description
+
+The verify service ensures that ingested files are encrypted with the correct key, and that the provided checksums match those of the ingested files.
+
+When running, verify reads messages from the configured RabbitMQ queue (default: "archived").
+For each message, these steps are taken (if not otherwise noted, errors halts progress and the service moves on to the next message.
```

Comment:
```suggestion
For each message, these steps are taken (if not otherwise noted, errors halt progress and the service moves on to the next message.
```